config/vite/plugin-mastodon-themes.ts

```diff
@@ -54,5 +39,116 @@ export function MastodonThemes(): Plugin {
         },
       };
     },
+    async configureServer(server) {
+      const themes = await loadThemesFromConfig(projectRoot);
```

Comment:
This would still be loaded only on boot, so I would move that to the middleware itself a few lines below.